lirc/__init__.py

```diff
@@ -26,15 +26,15 @@
 
 from lib.model.smartplugin import SmartPlugin
 
-REMOTE_ATTRS = ['lirc_instance', 'lirc_remote', 'lirc_key']
+REMOTE_ATTRS = ['lirc_remote', 'lirc_key']
 class LIRC(lib.connection.Client,SmartPlugin):
 
     ALLOW_MULTIINSTANCE = True
     PLUGIN_VERSION = "1.3.1"
 
     def __init__(self, sh, lirc_Host, lirc_Port=8765):
```

Comment:
Does it matter using uppercase in arguments? The `plugin.yaml` named them in lowercase.